Tests fail when MAPBOX_ACCESS_TOKEN is set

```
        result = runner.invoke(
            main_group,
            ['geocoding', '--reverse'],
            input='{0},{1}'.format(lon, lat),
            env={'MapboxAccessToken': 'bogus'})
>       assert result.exit_code == 0
E       assert -1 == 0
E        +  where -1 = <Result ConnectionError(u'Connection refused: GET https://api.mapbox.com/geoco...123xyz',)>.exit_code

tests/test_geocoding.py:93: AssertionError
```
where `MAPBOX_ACCESS_TOKEN=123xyz` is set in my environment. If I `unset MAPBOX_ACCESS_TOKEN` it works. 

@sgillies is this just a matter of `MAPBOX_ACCESS_TOKEN` taking precedence over `MapboxAccessToken`?

--- a/tests/test_config.py
+++ b/tests/test_config.py
@@ -43,11 +43,12 @@ def test_config_envvar(monkeypatch):
 def test_config_envvar_2(monkeypatch):
     """Get access token from MapboxAccessToken."""
     monkeypatch.setenv('MapboxAccessToken', 'pk.test_config_envvar_2')
+    monkeypatch.delenv('MAPBOX_ACCESS_TOKEN')
     runner = CliRunner()
     result = runner.invoke(main_group, ['config'], catch_exceptions=False)
     assert "Config file" not in result.output
-    assert "access-token = pk.test_config_envvar_2" in result.output
     assert "MapboxAccessToken = pk.test_config_envvar_2" in result.output
+    assert "access-token = pk.test_config_envvar_2" in result.output
     monkeypatch.undo()
 
 

--- a/tests/test_config.py
+++ b/tests/test_config.py
@@ -43,7 +43,7 @@ def test_config_envvar(monkeypatch):
 def test_config_envvar_2(monkeypatch):
     """Get access token from MapboxAccessToken."""
     monkeypatch.setenv('MapboxAccessToken', 'pk.test_config_envvar_2')
-    monkeypatch.delenv('MAPBOX_ACCESS_TOKEN')
+    monkeypatch.delenv('MAPBOX_ACCESS_TOKEN', raising=False)
     runner = CliRunner()
     result = runner.invoke(main_group, ['config'], catch_exceptions=False)
     assert "Config file" not in result.output

--- a/tests/test_geocoding.py
+++ b/tests/test_geocoding.py
@@ -42,7 +42,7 @@ def test_cli_geocode_fwd_env_token():
     result = runner.invoke(
         main_group,
         ['geocoding', '--forward', '1600 pennsylvania ave nw'],
-        env={'MapboxAccessToken': 'bogus'})
+        env={'MAPBOX_ACCESS_TOKEN': 'bogus'})
     assert result.exit_code == 0
     assert result.output == '{"query": ["1600", "pennsylvania", "ave", "nw"]}\n'
 
@@ -89,7 +89,7 @@ def test_cli_geocode_reverse_env_token():
         main_group,
         ['geocoding', '--reverse'],
         input='{0},{1}'.format(lon, lat),
-        env={'MapboxAccessToken': 'bogus'})
+        env={'MAPBOX_ACCESS_TOKEN': 'bogus'})
     assert result.exit_code == 0
     assert result.output.strip() == body
 

